build.gradle

```diff
@@ -52,6 +52,10 @@ dependencies {
     // Gson
     implementation 'com.google.code.gson:gson:2.9.1'
 
+    //spring
+    implementation 'org.springframework:spring-webmvc:5.3.20'
```

Comment:
By having these dependencies in the project, it means that the driver is tied with the spring. We don't want people to be forced to use spring.

It makes sense for such repository code to be in a separate module, that the spring users can include, but the non-spring users don't need to include.